app/views/plantings/_card.html.haml

```diff
@@ -37,22 +37,4 @@
           %dd= planting.finish_predicted_at if planting.finish_predicted_at.present?
 
     %p= render 'plantings/progress', planting: planting, show_explanation: true
-
-    = link_to 'Details', planting_path(planting),
-      class: 'btn btn-default btn-xs'
-
-    - if can?(:edit, planting) && can?(:create, Harvest)
```

Comment:
The `can?(:create, Harvest)` check is missing from _actions.html.haml - is it unnecessary, or should it be ported across?